web/frontend/src/components/keyboard-input.vue

```diff
@@ -44,11 +48,11 @@ const normalizeKey = (key: string): Keys | null => {
 };
 
 const emitKeyDown = (key: Keys) => {
-  if (!isActive) {
-    // eslint-disable-next-line no-use-before-define
-    toggleKeyboard(true);
-  }
   pressedKeys[key] = true;
+  if (!props.tempDisable) {
```

Comment:
Why are we still allowing the key to be registered in `pressedKeys` when disabled?

Reply:
Not sure. Fixing.